src/http/input/metadata/ContentTypeParser.ts

```diff
@@ -11,7 +11,7 @@ export class ContentTypeParser extends MetadataParser {
   public async handle(input: { request: HttpRequest; metadata: RepresentationMetadata }): Promise<void> {
     const contentType = input.request.headers['content-type'];
     if (contentType) {
-      input.metadata.contentType = parseContentType(contentType).type;
+      input.metadata.contentType = parseContentType(contentType).value;
```

Comment:
```suggestion
      input.metadata.contentType = contentType;
```

At this point we can just do this right? It should still throw the same errors and not change anything for the other classes, besides that they can now also request the extra parameters if they want to.

Reply:
If you do it like that, a header of `text/plain; charset=utf-8` would be returned.
The way i did it, it would return `text/plain`

Reply:
`ContentTypeParser` is the one that takes the content-type header of the request and converts it to metadata so it can be used internally. So we want to fully parse it.

Creating the content-type header of the response happens in the `MappedMetadataWriter` where the object of the format triple is inserted into the content-type header.

And I actually would want that response to be `text/plain; charset=utf-8` if that is the content-type that we're storing, but that would require a new metadata writer so maybe something for a different PR.